NodeJS/src/RestClient.ts

```diff
@@ -78,12 +81,11 @@ export class RestClient {
     public getSubAccount() {
         let url = `${this.baseUrl}/${USER_PATH}`
         logger.debug(`getSubAccount url is ${url}`)
+        console.log(`getSubAccount url is ${url}`)
         const options = this.prepareGetOptions(url)
         return axios(options)
             .then((response: { data: { [x: string]: any; }; }) => {
-                //logger.debug(response);
                 let accessibleSubAccounts = response.data['accessibleClients'];
-                // logger.debug(accessibleSubAccounts);
                 accessibleSubAccounts = lodash.sortBy(accessibleSubAccounts, ['id'])
                 let subAccount = accessibleSubAccounts[0]
                 return subAccount
```

Comment:
just return accessibleSubAccounts[0]; instead of assigning it to variable and then return because there is no use of this variable.